src/main/java/com/ma/springboot/service/GenericService.java

```diff
@@ -0,0 +1,11 @@
+package com.ma.springboot.service;
+
+import java.util.List;
+import java.util.Set;
+
+public interface GenericService<T> {
```

Comment:
```suggestion
public interface EntityService<T> {
```
Можливо так? Тоді стає зрозуміліше що буде працювати з моделями. Бо GenericService не зовсім зрозуміло в чому саме Generic